cockatrice/src/userlist.cpp

```diff
@@ -130,6 +133,8 @@ WarningDialog::WarningDialog(const QString userName, const QString clientID, QWi
     warningOption = new QComboBox();
     warningOption->addItem("");
 
+    deleteMessages = new QCheckBox(tr("Redact all messages from this user from rooms"));
```

Comment:
"from rooms" => "in all rooms" (why duplicate?)

Reply:
one of these is for when you warn them, the other is for when you ban them

Reply:
```suggestion
    deleteMessages = new QCheckBox(tr("Redact all messages from this user in all rooms"));
```